codegen/tests/cardspb/cards.pb.go

```diff
@@ -5,9 +5,8 @@ package cardspb
 
 import (
 	fmt "fmt"
-	math "math"
-
 	proto "github.com/golang/protobuf/proto"
+	math "math"
```

Comment:
i have always run `goimports` on the generated file, which is what gives the original ordering you had. I think it is better to do that. maybe.

Reply:
Done.